src/core/tiledscene/qgstiledscenetile.h

```diff
@@ -164,6 +164,20 @@ class CORE_EXPORT QgsTiledSceneTile
      */
     void setBaseUrl( const QUrl &baseUrl );
 
+    /**
+     * Returns the up axis to use for glTF resources associated with the tile.
+     *
+     * \see setGltfUpAxis()
+     */
+    Qgis::Axis gltfUpAxis() const;
+
+    /**
+     * Sets the up \a axis to use for glTF resources associated with the tile.
+     *
+     * \see gltfUpAxis()
+     */
+    void setGltfUpAxis( Qgis::Axis axis );
```

Comment:
Hmm I am not very fond of having this exposed in QgsTiledSceneTile - it looks like some legacy pre-1.0 3d-tiles stuff, I assume other tiled scene data providers would not even use gltf... What do you think about having the axis orientation passed as some provider-specific metadata, e.g. `QVariantMap metadata() const;` and the 2d/3d renderers would consult that?

Reply:
Ok, done!